docs/dataset.md

```diff
@@ -2,28 +2,19 @@
 
 Promscale stores some configuration information in the Postgres database which it is connected to. We call this configuration the *Promscale dataset configuration*. Promscale accepts an option to set the dataset values. This document describes its format and mechanics.
 
-Setup is done by using the `-startup.dataset.config` flag which should contain this configuration structure in YAML format
-
-Example usage:
-
-```bash
-promscale -startup.dataset.config=$'metrics:\n  default_chunk_interval: 6h'
-```
-
-The expected format is YAML.
-
-The YAML values are easier to set when using `config.yaml` instead of the cli flags. This would look as follows:
+Setup is done via the `config.yaml` under `startup.dataset`:
 
 ```yaml
-startup.dataset.config: |
-  metrics:
-    default_chunk_interval: 6h
-    compress_data: true
-    ha_lease_refresh: 10s
-    ha_lease_timeout: 1m
-    default_retention_period: 90d
-  traces:
-    default_retention_period: 30d
+startup:
+  dataset:
+    metrics:
```

Comment:
I know its not a part of this PR but could you update the table below to call the section `metrics` instead of `metric`?